lgc/unittests/interface/OptLevelTest.cpp

```diff
@@ -41,7 +41,15 @@ TEST(LgcInterfaceTests, DefaultOptLevel) {
   unsigned palAbiVersion = 0xFFFFFFFF;
   StringRef gpuName = "gfx802";
 
+#if LLVM_MAIN_REVISION && LLVM_MAIN_REVISION < 474768
+  // Old version of the code
   for (auto optLevel : {Level::None, Level::Less, Level::Default, Level::Aggressive}) {
+#else
+  // New version of the code (also handles unknown version, which we treat as latest)
+  // Returns the optimization level for the context.
+  for (auto optLevel :
+       {CodeGenOptLevel::None, CodeGenOptLevel::Less, CodeGenOptLevel::Default, CodeGenOptLevel::Aggressive}) {
+#endif
```

Comment:
"endif" colouring looks off. Not sure if this is a major issue, but this can be avoided by not placing an opening bracket in the if-def (or placing both brackets in the ifdef, but that would mean more code duplication).

Reply:
Do you mean the '{' for the for?
I wonder if clang-format will complain if I did that.
I propose to leave this as-is.